tensorflow/core/common_runtime/gpu/process_state.cc

```diff
@@ -181,11 +181,23 @@ Allocator* ProcessState::GetCUDAHostAllocator(int numa_node) {
   // different numa_nodes.  For now, just one.
   numa_node = 0;
   mutex_lock lock(mu_);
+
+  // CUDAHost alloc the same across all gpus, so just get the executor for the
```

Comment:
Maybe start this with "Any valid stream executor works for allocating CUDA host memory, so just ..."